### ISSUE TITLE
Read concepts from text file in fathomnet_generate

### ISSUE THREAD

**Issue by eor314** [fathomnet/fathomnet-py#3]
Instead of reading concepts from the command line, allow users to point the CLI to a text file. Would be helpful for big groups of concepts.

**Comment by kevinsbarnard** [comment]
Got it. I'll tentatively plan on a newline-delimited list of concepts:

`example.txt`:
```
Apolemia
Abraliopsis
Chionoecetes tanneri
```

with usage

```bash
fathomnet-generate --concept-file example.txt ...
```

**Comment by kevinsbarnard** [comment]
Resolved by #4
